packages/webapp/pages/posts/[id]/edit.tsx

```diff
@@ -31,6 +32,8 @@ import { getLayout as getMainLayout } from '../../../components/layouts/MainLayo
 import { defaultOpenGraph, defaultSeo } from '../../../next-seo';
 
 function EditPost(): ReactElement {
+  gqlClient.unsetHeader('content-language');
```

Comment:
but would it then not unset the language for all further requests?

Reply:
Nope, doesn't appear to do so. I've tested by selecting Spanish. Browse my feed and see posts are in Spanish, go to a post I can edit and see it fetches in English, then navigate to popular feed (to force fetching of something new) and see posts are in Spanish.

Reply:
Because in BootProvider it will set back on any render and then EditPage renders lower (as child) and unsets it, so yeah this works but it could be flaky in some cases, would prefer we update gql client at some point so we can inject header with requests as well. We will get to it hopefully 🤞